This patch fixes the section of the rubber info page (teTeX -> TeX).
The section teTeX does not exist any more (#424946).
Index: rubber-1.1+20100306/doc/rubber.texi
===================================================================
--- rubber-1.1+20100306.orig/doc/rubber.texi	2012-06-25 18:41:10.000000000 +0200
+++ rubber-1.1+20100306/doc/rubber.texi	2012-08-08 14:13:37.000000000 +0200
@@ -42,7 +42,7 @@
 
 @c ---  Introduction  ---
 
-@dircategory teTeX
+@dircategory TeX
 @direntry
 * Rubber: (rubber).   A building system for LaTeX documents.
 @end direntry
